libraries/botbuilder/src/teamsActivityHandler.ts

```diff
@@ -779,7 +821,7 @@ export class TeamsActivityHandler extends ActivityHandler {
      * @param context A context object for this turn.
      * @returns A promise that represents the work queued.
      */
-    protected async onTeamsChannelDeleted(context): Promise<void> {
+    protected async onTeamsChannelDeleted(context: TurnContext): Promise<void> {
```

Comment:
I added this because types strict mode isn't on. I plan on incremental changes to properly utilize TS.